### ISSUE TITLE
[ADE Support] Poll accessibility specialists CoP to prioritize potential research topics

### ISSUE THREAD

**Issue by cindymerrill** [department-of-veterans-affairs/va.gov-team#105552]
> [!NOTE]
> **QUICK SUMMARY:** _Create a poll to get prioritization feedback from the accessibility specialist community of practice_

## Description of the Work
I want to get feedback from accessibility specialists on which user research topics would be the most useful for helping advise VFS teams about accessibility so that I can work on the most impactful research. 

### Background
I have a spreadsheet of potential accessibility research topics that have been suggested recently, and I’m looking for help with prioritizing. I want to know which topics would be the most useful.

- **Slack Channel:** Planning to launch in DSVA accessibility-specialists

## Supporting Artifacts 
- [Spreadsheet of accessibility research ideas](https://docs.google.com/spreadsheets/d/1ONts6e7kBKFooBZ_i-d7Dk_75F_oMeOKAX5vwHQu3io/edit?usp=sharing)

## ADE Tasks
- [x] Draft poll
- [x] Post poll in team channel in agency slack for feedback
- [x] Incorporate feedback
- [x] Post poll in team channel in OCTO internal slack for feedback
- [x] Incorporate feedback
- [x] Post poll in OCTO accessibility-specialists channel
- [x] Summarize responses

## Acceptance Criteria
- [x] Poll posted in OCTO accessibility-specialists channel
- [x] Poll results summarized and posted on slack
- [x] Research ideas updated with poll results

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] **Read** Important Notifications (ADE Internal) at the bottom of this ticket.
- [ ] **Delete** Important Notifications (ADE Internal) at the bottom of this ticket, after reading. 

<HR> 

> **Last updated:** 2/25/2025

**Comment by cindymerrill** [comment]
Revised emoji poll with feedback and then posted again for feedback in our agency slack team channel, along with a followup question. 

Then I posted [here](https://dsva.slack.com/archives/C08B367JLLT/p1742421384574729) in our OCTO internal slack for Martha to see before I post in the accessibility-specialists channel.

**Comment by cindymerrill** [comment]
Here's the summary of the research topic poll: 

-   Errors – 3 votes
-   Progress bar (segmented) – 3 votes
-   Alerts – 3 votes
-   Content and overwhelm – 2 votes
-   Getting oriented to a new page – 1 vote
-   Breadcrumbs – 1 vote

I posted these results in the [User Research Ideas canvas](https://dsva.slack.com/docs/T03FECE8V/F08JZ9H5Y9X) and mentioned them in today's VA Accessibility Specialist Sync meeting. 

Closing ticket because the poll is done.
